shared/cluster.go

```diff
@@ -655,3 +656,64 @@ func PrintGetAll() {
 	fmt.Printf("\n\n\n-----------------  Results from kubectl get all -A -o wide"+
 		"  -------------------\n\n%v\n\n\n\n", res)
 }
+
+func CreateSecret(secret, namespace string) error {
+	kubectl := fmt.Sprintf("kubectl --kubeconfig %s", KubeConfigFile)
+	if namespace == "" {
+		namespace = "default"
+	}
+	cmd := fmt.Sprintf("%s create secret generic %s -n %s --from-literal=mykey=mydata",
+		kubectl, secret, namespace)
```

Comment:
can we add a break line here? lets try at least on shared stuff maintain code blocks with context of usage and etc , gives a better and clear reading of the code.